src/core/account_manager.mjs

```diff
@@ -126,20 +122,18 @@ export class AccountManager {
    * batch up the accounts into sets to be processed
    * @returns {*[]} an array of arrays of numbers representing the accounts to update
    */
-  batchAccounts () {
+  batchAccounts (accountRange = constants.SYSTEM_ACCOUNTS) {
     const batchSize = constants.ACCOUNT_CREATE_BATCH_SIZE
     const batchSets = []
 
-    let batchCounter = 0
     let currentBatch = []
-    for (const [start, end] of constants.SYSTEM_ACCOUNTS) {
+    for (const [start, end] of accountRange) {
+      let batchCounter = start
       for (let i = start; i <= end; i++) {
         currentBatch.push(i)
         batchCounter++
 
-        // push only a single account for the first to give time for the
-        // network to create all the system accounts and become stable
-        if (batchCounter >= batchSize || batchSets.length < 1) {
```

Comment:
I added `batchSets.length < 1` because I noticed that the Hedera Services code is lazy initializing all of the system accounts when the first account update or create comes through.  If a batch of transactions hit it at the same time, it occasionally locks up the nodeClient and cause timeouts.  Has this been running successfully both locally and in the pipeline consistently?

Reply:
I see you made it contiguous, which would add a lot of stability at the cost of speed.  I'm guessing we won't need the account key update very often, so it is worth it?

Reply:
If you fixed the root cause with the logger in the hedera-sdk library, then why wouldn't we want to keep the asynchronous code?  It would run a lot faster.  Is it still having issues, or did you make it synchronous before figuring out the root cause?

Reply:
> did you make it synchronous before figuring out the root cause?

Good question, I don't remember exactly. My original suspicion was that same _nodeClient was being used to invoke transactions in "parallel" and probably some internal state was causing trouble. 

Sequential operation made it a lot simpler to pinpoint the hedera-sdk bug. So far, it is very stable.

We can reintroduce the batch and see how stable that is.

Reply:
I've reintroduced the parallel account update. It reduced the execution time significantly (32 seconds compared to >8minutes). There was a spike of open sockets (from 120 to 260) as expected, but such as spike of few hundred sockets is ok I think.

<img width="649" alt="Screenshot 2024-03-19 at 2 30 03 PM" src="https://github.com/hashgraph/solo/assets/811437/5c768bc0-6da1-4896-b4f6-5addb7cc8d36">